ImportError: No module named 'configuration'

I am following the instruction, but when I try to build I get 

Traceback (most recent call last):
  File "extras/pokemontools/scan_includes.py", line 33, in <module>
    dependencies += recursive_scan(os.path.join(conf.path, filename))
  File "extras/pokemontools/scan_includes.py", line 25, in recursive_scan
    includes = recursive_scan(os.path.join(conf.path, include), includes)
  File "extras/pokemontools/scan_includes.py", line 16, in recursive_scan
    lines = open(filename).readlines()
  File "/usr/lib/python3.4/codecs.py", line 313, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe9 in position 11: invalid continuation byte
Traceback (most recent call last):
  File "prequeue.py", line 11, in <module>
    import preprocessor
  File "/home/manolis/git_repos/pokecrystal/preprocessor.py", line 6, in <module>
    import extras.pokemontools.configuration as configuration
  File "/home/manolis/git_repos/pokecrystal/extras/pokemontools/__init__.py", line 1, in <module>
    import configuration as config
ImportError: No module named 'configuration'
Makefile:58: recipe for target 'wram.o' failed
make: *** [wram.o] Error 1

I tried with python3.4 and python 3, gcc 4.9.0

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -1,5 +1,7 @@
 # Linux
 
+Note that this requires python2.7 for the moment.
+
 	sudo apt-get install make gcc bison git python python-setuptools
 
 	git clone git://github.com/bentley/rgbds.git

